setup.py

```diff
@@ -27,4 +44,13 @@
     # For an analysis of "install_requires" vs pip's requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
```

Comment:
```suggestion
    # https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/
```
The above link it dead.

Reply:
I fixed it.